google-cloud-core/src/main/java/com/google/cloud/Timestamp.java

```diff
@@ -189,8 +189,8 @@ public static Timestamp fromProto(com.google.protobuf.Timestamp proto) {
   }
 
   /**
-   * Creates a Timestamp instance from the given string. String is in the RFC 3339 format without
-   * the timezone offset (always ends in "Z").
+   * Creates a Timestamp instance from the given string. String is in the RFC 3339 format with
+   * timezone offset or without timezone offset.
```

Comment:
'with timezone offset or without timezone offset' does not bring any information to user, you can remove it.
Instead you can provide the examples the method can recognize. This would be really helpful.

Reply:
done